web/src/navbar_alerts.ts

```diff
@@ -190,6 +203,39 @@ const DESKTOP_NOTIFICATIONS_BANNER: AlertBanner = {
     custom_classes: "navbar-alert-banner",
 };
 
+const bankruptcy_banner = (): AlertBanner => {
+    const old_unreads_missing = unread.old_unreads_missing;
+    const unread_msgs_count = unread.get_unread_message_count();
+    return {
+        process: "bankruptcy",
+        intent: "info",
+        label: $t(
+            {
+                defaultMessage:
+                    "Welcome back! You have {old_unreads_missing, select, true {at least} other {}} {unread_msgs_count} unread messages. Do you want to mark them all as read?",
```

Comment:
I'm not sure Transifex fully supports this i18n syntax. Generally I'd prefer to have the conditional logic visible in the code as an if/else statement before this `return` statement anyway.

Reply:
I have removed this i18n syntax from the code, and replaced it with a conditional logic instead just before the `return` statement.